packages/nouns-webapp/src/index.tsx

```diff
@@ -60,28 +61,24 @@ const createRootReducer = () =>
     onDisplayAuction,
   });
 
-export function configureStore(preloadedState: PreloadedState<any>) {
-  return createStore(
-    createRootReducer(), // root reducer without router state
-    preloadedState,
-    composeWithDevTools(
-      applyMiddleware(),
-      // ... other middlewares ...
-    ),
-  );
-}
+const loggerMiddleware = createLogger();
 
-const store = configureStore({});
+export const store = configureStore({
+  reducer: createRootReducer(),
+  middleware: getDefaultMiddleware => getDefaultMiddleware().concat(loggerMiddleware),
```

Comment:
can we please conditionally enable it with a ENV?
this makes the console too noisy...its harder to debug other stuff

Reply:
It's done.